src/Symfony/Bridge/PhpUnit/DnsMock.php

```diff
@@ -30,6 +30,7 @@ class DnsMock
         'NAPTR' => \DNS_NAPTR,
         'TXT' => \DNS_TXT,
         'HINFO' => \DNS_HINFO,
+        'CAA' => \DNS_CAA,
```

Comment:
```suggestion
        'CAA' => '\\' !== \DIRECTORY_SEPARATOR ? \DNS_CAA : 0;
```

Reply:
Thank you @nicolas-grekas for the suggestion! 🙏